runtime/compiler/codegen/J9CodeGenerator.cpp

```diff
@@ -3917,6 +3917,53 @@ J9::CodeGenerator::collectSymRefs(
    }
 
 
+
+void
+J9::CodeGenerator::fixUpProfiledInterfaceGuardTest()
```

Comment:
Can we please Doxygen document precisely what this phase does? Let's take this opportunity to clean up the triple newline immediately before as well.